src/build.netcore.props

```diff
@@ -3,7 +3,8 @@
   <Import Project="$(MSBuildThisFileDirectory)build.common.props" />
 
   <PropertyGroup>
-    <NetCoreVersion>netcoreapp3.1</NetCoreVersion>
+    <NetSixVersion>net6.0</NetSixVersion>
+    <NetCoreVersion>netcoreapp3.1</NetCoreVersion> 
```

Comment:
>NetCoreVersion [](http://example.com/codeflow?start=5&length=14)

I don't think you need a new property, just set the following value to NetCoreVersion?

<NetCoreVersion>netcoreapp3.1;net6.0</NetCoreVersion>